app/datadog-init.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import { datadogRum } from "@datadog/browser-rum";
+
+
+console.log("Datadog Init", process.env.NEXT_PUBLIC_DATADOG_RUM_APPLICATION_ID, process.env.NEXT_PUBLIC_DATADOG_RUM_CLIENT_TOKEN);
```

Comment:
Why keeping this?

Reply:
ah forgot to remove, thanks for pointing out